pyproject.toml

```
@@ -1,6 +1,90 @@
+[project]
+name = "asv"
+description = "Airspeed Velocity: A simple Python history benchmarking tool"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+license = { file = "LICENSE.rst", content-type = "text/plain" }
```

> Is this the equivalent of "BSD 3 clause licence"?

Good catch; it appears that the `license` key can either specify a file OR a text string, so since the classifier doesn't specify `BSD 3 Clause` it's probably better to specify it as text here:
```suggestion
license = { text = "BSD-3-Clause" }
```